sql/src/SqlDate.sk

```diff
@@ -14,7 +14,7 @@ module SKDB;
  *  localtime       | adjusts time value to its left to the local time
  *  utc             | adjusts time value to its left to the utc time
  *  tz <timezone>   | adjusts time value to its left to the specified timezone (<timezone>: ±D | ±DD | ±DDDD | ±DD:DD | ±DD:DD:DD)
- *  isutc           | force the time value to its left to be in utc time ingnoring current timezome
+ *  islocal         | force the time value to its left to be in local time ignoring current timezome
```

Comment:
```suggestion
 *  islocal         | force the time value to its left to be in local time
```

Reply:
Lol, I was distracted by the typo and missed the bigger picture.